.ci/validation.sh

```diff
@@ -411,20 +411,21 @@ checkstyle-and-sevntu)
     -Dpmd.skip=true -Dspotbugs.skip=true -Djacoco.skip=true
   ;;
 
-spotbugs-and-pmd)
-  mkdir -p .ci-temp/spotbugs-and-pmd
+spotbugs-pmd-rewrite)
```

Comment:
Please keep it separate. 
It most likely be very opinionated and conflicting with others, it should be separate job in CI to let clearly see who is reason of failure.